source/user-manual/reference/tools/rbac-control.rst

```diff
@@ -0,0 +1,102 @@
+.. Copyright (C) 2022 Wazuh, Inc.
+
+.. _rbac_control:
+
+rbac_control
+============
+
+.. versionadded:: 4.5.0
+
+The ``rbac_control`` tool allows changing the default users' password and reseting the RBAC database to its default
+state. For more information about the Wazuh RBAC resources and database, please visit the
+:ref:`How it works <api_rbac_how_it_works>` section.
+
+Usage
+-----
+
++-----------------------------------------+---------------------------------------------------+
+| Option name                             | Option description                                |
++=========================================+===================================================+
+| ``-h, --help``                          | Display the help message.                         |
++-----------------------------------------+---------------------------------------------------+
+| ``change-password``                     | Change the password for each default user.        |
++-----------------------------------------+---------------------------------------------------+
+| ``factory-reset``                       | Reset the RBAC database to its default state.     |
```

Comment:
Look for a way to add that this action requires a confirmation message unless the `-f` or `--force` flag is added.